bin/vmware-host-health.rb

```diff
@@ -0,0 +1,185 @@
+#! /opt/sensu/embedded/bin/ruby
+
+require 'sensu-plugin/check/cli'
+require 'rbvmomi'
+require 'json'
+
+class EsxiCheck < Sensu::Plugin::Check::CLI
+  check_name nil
+  option :host,
+         short: '-H HOST',
+         long: '--host HOST',
+         default: 'localhost',
+         description: 'ESXi host'
+
+  option :verify_ssl,
+         description: 'Turn on/off using SSL certificate (default: false)',
+         short: '-v',
+         long: '--verify_ssl',
+         boolean: true,
+         default: false
+
+  option :username,
+         short: '-u USERNAME',
+         long: '--username USERNAME',
+         default: 'root',
+         description: 'API username'
+
+  option :password,
+         short: '-p PASSWORD',
+         long: '--password PASSWORD',
+         default: 'root',
+         description: 'API password'
+
+  option :dryrun,
+         short: '-d',
+         long: '--dryrun',
+         description: 'run without sending to sensu but see what would be sent',
+         boolean: true,
+         default: false
+
+  option :handlers,
+         description: 'Comma separated list of handlers',
+         long: '--handlers <HANDLER>',
+         proc: proc { |s| s.split(',') },
+         default: []
+
+  option :help,
```

Comment:
pretty sure we get this for free without needing to do anything:
```
$ ./bin/check-cpu.rb --help
Usage: ./bin/check-cpu.rb (options)
        --cache-file CACHEFILE
    -c CRIT
        --guest                      Check cpu guest instead of total cpu usage
        --guest_nice                 Check cpu guest_nice instead of total cpu usage
        --idle                       Check cpu idle instead of total cpu usage
        --idle-metrics METRICS       Treat the specified metrics as idle. Defaults to idle,iowait,steal,guest,guest_nice
        --iowait                     Check cpu iowait instead of total cpu usage
        --irq                        Check cpu irq instead of total cpu usage
    -l, --less_than                  Change whether value is less than check
        --nice                       Check cpu nice instead of total cpu usage
        --proc-path /proc
        --sleep SLEEP                This sleep controls the interval between the initial poll for cpu utilization and the next data point, the longer the interval is the more accurate your data will be
        --softirq                    Check cpu softirq instead of total cpu usage
        --steal                      Check cpu steal instead of total cpu usage
        --system                     Check cpu system instead of total cpu usage
        --user                       Check cpu user instead of total cpu usage
    -w WARN

$ grep -rc help ./bin/check-cpu.rb 
0
```
Was there some default setting you wanted changed?

Reply:
if we do ill remove it. Its a force of habbit